python/mxnet/contrib/onnx/mx2onnx/export_model.py

```diff
@@ -43,8 +43,8 @@ def export_model(sym, params, input_shape, input_type=np.float32,
         Path to the params file or params dictionary. (Including both arg_params and aux_params)
     input_shape : List of tuple
         Input shape of the model e.g [(1,3,224,224)]
-    input_type : data type
-        Input data type e.g. np.float32
+    input_type : List of dtype
```

Comment:
We can say data type or list of data types

Reply:
done